### PR TITLE
API stability - phase 1

### PR DESCRIPTION
**Pull request by YusukeHosonuma**
Note: phase 2 will another PR that include `Option` API changes.

## TODO
- [x] change API
- [x] add documentation comments
- [x] update README
- [x] update Example app

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 ![Logo](https://raw.githubusercontent.com/YusukeHosonuma/SwiftPrettyPrint/master/Image/logo.png)
 
-Pretty print for debug that readable output than `print()` and `debugPrint()`.
+Pretty print that **Human-readable output** than `print()` and `debugPrint()` in Swift standard library.
```
Comment (sahara-ooga) [inline README.md]:
```suggestion
Pretty print that is **Human-readable output** than `print()` and `debugPrint()` in Swift standard library.
```

That clause needs verb. In this case, be verb seems suitable for me.

File: Sources/API/Debug.swift [hunk 2]
```diff
@@ -7,26 +7,78 @@ public class Debug {
...
 
-// MARK: Print to console
+// MARK: Standard API
 
 extension Debug {
-    public static func p<T>(_ target: T, debug: Bool = false) {
-        print(pString(target, debug: debug))
+    /// Output `target` to console.
+    /// - Parameter target: target
+    /// - Returns: String that same of output console.
```
Comment (sahara-ooga) [inline Sources/API/Debug.swift]:
```suggestion
    /// - Returns: String that is the same as output console.
```

Comment (sahara-ooga) [reply Sources/API/Debug.swift]: That clause needs a verb in it. 
And "same" is used as adverb ahead of noun, or pronoun. This suggestion adopts the latter usage.

File: Sources/API/Debug.swift [hunk 3]
```diff
@@ -7,26 +7,78 @@ public class Debug {
...
     }
 }
 
-// MARK: Get as string
+// MARK: Alias API
 
 extension Debug {
-    public static func pString<T>(_ target: T, debug: Bool = false) -> String {
-        pretty.string(target, debug: debug, pretty: false)
+    /// Alias to `print()` and `debugPrint`
```
Comment (sahara-ooga) [inline Sources/API/Debug.swift]:
```suggestion
    /// Alias to `print()` and `debugPrint()`
```

File: Sources/API/Debug.swift [hunk 4]
```diff
@@ -7,26 +7,78 @@ public class Debug {
...
+        if debug {
+            return debugPrint(target)
+        } else {
+            return print(target)
+        }
     }
 
-    public static func ppString<T>(_ target: T, debug: Bool = false) -> String {
-        pretty.string(target, debug: debug, pretty: true)
+    /// Alias to `prettyPrint()` and `debugPrettyPrint`
```
Comment (sahara-ooga) [inline Sources/API/Debug.swift]:
```suggestion
    /// Alias to `prettyPrint()` and `debugPrettyPrint()`
```

**Review by sahara-ooga — APPROVED** [review]
I cite some minor suggestions about English grammar. But they don't have to be applied.
The changes in this PR are reasonable👍🏻.

I think some users hope the option that turns off debug console output (then, `Debug.print()` function only returns `String` as return value).

**Comment by YusukeHosonuma** [comment]
@sahara-ooga 
> I think some users hope the option that turns off debug console output (then, Debug.print() function only returns String as return value).

Thanks for your valuable suggestions.
I created issue #44 , I'll think API specification and implement in another PR probably.

**Review by po-miyasaka — APPROVED** [review]

**Pull request merged**